geode-protobuf/src/main/proto/connection_API.proto

```diff
@@ -16,6 +16,26 @@
 syntax = "proto3";
 package org.apache.geode.internal.protocol.protobuf;
 
+enum MajorVersions {
```

Comment:
A comment explaining what these are about would be nice: that current version will track the version that's deployed in the API. Could we write a test for this? Do we archive proto files somewhere? (maybe a problem for down the line)

Reply:
The real definitions of major and minor version are a little nebulous at that point, so I don't want to try to commit to anything at this point.

We do use the CURRENT_{MAJOR,MINOR}_VERSION throughout our tests, though we don't currently have any idea of other versions or how we might handle them (aside from outright failing version mismatches).